libcxx/test/std/containers/views/mdspan/mdspan.extents.cons/convertible_to_size_t.pass.cpp

```diff
@@ -0,0 +1,48 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <experimental/mdspan>
+#include <cassert>
+#include <array>
+#include <span>
```

Comment:
```suggestion
#ifdef __cpp_lib_span
#include <span>
#endif
```
I don't think you can even include `<span>` unless that feature test macro is defined.

Reply:
Thanks Mark, you are right.  However, I don't think an inclusion of `<span>` is even needed here, since it's already included by `<mdspan>`.  Tests ran fine without this line.
Actually, `#include <array>` is not needed here either.

Reply:
Yeah I'm aware of that.  And any test that uses `span` should be guarded by `#ifdef __cpp_lib_span` as well (which has been fixed).